src/pages/Delivery/hooks/useCampusDeliveryAddress.ts

```diff
@@ -0,0 +1,15 @@
+import { useSuspenseQuery } from '@tanstack/react-query';
+import { getCampusDeliveryAddress } from '@/api/delivery';
+import { CampusDeliveryAddressRequest } from '@/types/api/deliveryCampus';
+
+const useCampusDeliveryAddress = ({ filter }: CampusDeliveryAddressRequest) => {
+  return useSuspenseQuery({
+    queryKey: ['campusDeliveryAddress', filter],
+    queryFn: async () => {
+      const { count, addresses } = await getCampusDeliveryAddress({ filter });
+      return { count, addresses };
```

Comment:
```suggestion
    queryFn: () => getCampusDeliveryAddress({ filter }),
```
여기도 별도로 구조 분해 할당 하지 않아도 괜찮을 것 같은데 하신 이유가 있을까요?

Reply:
data를 더 쪼개서 보낸다는 걸 잘못 거기서 해버렸네요. 수정했습니다!